### ISSUE TITLE
Skip link should go to H1

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#27445]
## User story
As a Veteran who uses assistive technology, orienting myself on websites is often unpredictable, cumbersome and time-consuming. 

## Issue Description
According to a recent Webaim survey, ~ 50% of screen readers use  ‘skip to content’ to navigate a webpage at least sometimes (if not always)

**Current experience:** On Facility Locator, Skip to content goes to the page breadcrumb which causes a screen reader user to have to manually skip over less important content before starting their search.

**Reference:** [Facility Locator short-term recommendations #2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/user-research/screenreader-usability-study/research-findings.md#on-facility-locator---short-term)

Detail page is being addressed in #25849 

## Accessibility notes
When the user uses Skip to content, the Page H1 should receive focus using a tabindex="-1" in order to be announced to screen reader users when landing on the page

## Dev Notes

- This will require a global change for React pages:
  - For vets-website: `src/platform/landing-pages/dev-template.ejs`
  - For static pages: `src/site/includes/header.html`
  - move div id="content" to new position, similar to Pittsburgh
- gov.uk has a live example for this 

---
## Tasks
- [ ] Make ‘Skip to content’ link go to the page H1
- [ ] Coordinate with PW/platform will review PR
- [ ] Add cypress test

## Acceptance Criteria
- [ ] When a screen reader user chooses ‘Skip to content’ on the Facility Locator search page, (s)he is taken to the page H1
- [ ] Issue to be validated by Josh and/or Angela

**Comment by mmiddaugh** [comment]
@joshkimux @AngelaFowler82 This appears to be a global issue - do you know if this is occurring everywhere across VA.gov? We're trying to evaluate if we should be making a change in the Facilities products or if this should be implemented broadly (by VSP or PW?)

**Comment by joshkimux** [comment]
@mmiddaugh , there has been conversations on this globally on VA.gov. @1Copenut and @kelsonic have been involved in that I believe, and there was a ticket but I can't quite find it.
